otx/algorithms/classification/adapters/mmcls/models/pretrainers/byol.py

```diff
@@ -0,0 +1,210 @@
+"""BYOL (Bootstrap Your Own Latent) implementation for self-supervised learning.
+
+Original papers:
+- 'Bootstrap Your Own Latent: A New Approach to Self-Supervised Learning', https://arxiv.org/abs/2006.07733
+"""
+
+# Copyright (C) 2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+# pylint: disable=missing-module-docstring, too-many-instance-attributes, unused-argument, unnecessary-pass, invalid-name
+from collections import OrderedDict
+from typing import Any, Dict, Optional
+
+import torch
+import torch.distributed as dist
+from mmcls.models.builder import CLASSIFIERS, build_backbone, build_head, build_neck
+from mpa.utils.logger import get_logger
+from torch import nn
+
+logger = get_logger()
+
+
+@CLASSIFIERS.register_module()
```

Comment:
Can we revise the base folder name as classifiers rather than pretrainers? Otherwise, do you intend to name it?

Reply:
I newly named it as pretrainer because it doesn't look like classifier.
But if it could be against policy, I'll modify it.